OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChatMessageWorkshopServiceTests.cs

```diff
@@ -35,7 +35,8 @@ public class ChatMessageWorkshopServiceTests
         WorkshopId = Guid.NewGuid(),
     };
 
-    private IEntityRepository<Guid, ChatMessageWorkshop> messageRepository;
+    //private IEntityRepository<Guid, ChatMessageWorkshop> messageRepository;
```

Comment:
(Code style) Please try to avoid presence of commented-out code. It has either be some `//TODO:` or nothing

Reply:
Fixed. Thank you!